src/answers-core-re-exports.ts

```diff
@@ -0,0 +1,49 @@
+export {
+  AnswersConfig,
+  AnswersCore,
```

Comment:
Is there a reason we are exporting AnswersCore? I believe it's just an internal dependency of Answers Headless which doesn't need to be available publicly

Reply:
AnswersCore and BaseAnswersConfig are being exported because AnswersConfig links to them in its documentation. If they aren't exported, API extractor gives "unresolved link" warnings

Reply:
If you don't export AnswersCore, will the docs generate with just the "AnswersCore" text, or will it include a broken link?

Reply:
It removes the text entirely, so the docs say something like "The main configuration options for .", where it should say "AnswersCore" before the period with a link

Reply:
Ideally we wouldn't export `AnswersCore`, but since it breaks the docs, I think we should leave it in. I suppose we could remove the link from Answers Core so that we no longer need to export it, but that's out of scope